1_get_raw_data/src/get_raw_data.R

```diff
@@ -18,8 +20,11 @@ get_sample <- function(file){
   
 }
 
-get_sites <- function(file){
+get_sites <- function(ind_file){
 
+  gd_get(ind_file, type='xlsx')
+  file <- as_data_file(ind_file)
+  
```

Comment:
I felt like I was doing this enough that maybe `gd_get` and `gd_put` should return `as_data_file(ind_file)` (invisible)?

Reply:
oh, good call on `data_file <- gd_get()`! i've just added it and will make a PR soon.

less sure what should be returned by gd_put(). in this PR you use it as 
```
get_flow <- function(summary.flow, ind_file){
  ...
  gd_put(ind_file, data_file)
}
```
and
```
4_discharge/out/raw_flow.rds.ind:
    command: get_flow(summary_flow, target_name) 
```
in which case there's no need to return anything from `get_flow`. i think this is the common case, where a function that calls gd_put has a remake target that's a file written by gd_put (the ind_file), rather than an R object. what's the use case where you'd want to have `as_data_file(ind_file)` (or any other file name) at your fingertips right after calling gd_put?

Reply:
also, what's your take on having `gd_get` return a file path visibly vs invisibly? i'm not really sure which would be better or how to decide.

Reply:
Oh, yeah, you're right. You'll need to run `as_data_file(ind_file)` before each `gd_put`....
So this is what I need to do now:
```
  data_file <- as_data_file(ind_file)
  saveRDS(all_flow, file = data_file)
  gd_put(ind_file, data_file)
```
What about if the `type` in `gd_put` was "rds"...that it would behave above, but in 1 line? Something like:
```
gd_put(ind_file, type="rds", object=all_flow)
```
?

Reply:
I'm pretty indifferent on `return` vs. `invisible` generally. At this stage in the package, I don't think it matters one bit. If people have been using the package awhile, I'd recommend `invisible` because they might start getting a bunch of new junk in the console they weren't used to seeing.

Reply:
cool, good thoughts.
* i'm going to go with visible gd_get output for now, to help with discovering that it returns that value (and because there are no long-standing users).
* and i'm going to make an issue for considering adding the feature of having gd_put save a file for you - my main hesitation is that it puts scipiper on the hook for doing something that can be done in many ways (exporting an object), but agreed that it would help clean up code if we could provide that option for common file types.